src/shared/styles/sticky.module.scss

```diff
@@ -2,10 +2,16 @@
 
 .sticky-tabs-container {
   & :global(.tabs__header) {
-    z-index: 100;
+    z-index: 5000;
     position: sticky;
     top: 0;
-    background: $colour-sky-white;
+    background: rgba($colour-sky-white, 0.75);
+    backdrop-filter: blur(2px);
```

Comment:
Thought it looked cool, but can definitely be removed if we don't like it. It gives the glossy effect on the tabs within the entry pages